packages/core/src/utils/CoreHelperUtil.ts

```diff
@@ -181,5 +181,18 @@ export const CoreHelperUtil = {
 
       return v.toString(16)
     })
+  },
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  parseError(error: any): string {
```

Comment:
I guess we're using `any` since we don't know what the error will be here. Would `unknown` work instead of `any`, so we wouldn't have to override ESLint rules?